### ISSUE TITLE
Strange search results

### ISSUE THREAD

**Issue by craigsapp** [TassoInMusicProject/tasso-website#115]
<img width="1178" alt="screen shot 2018-09-26 at 4 16 52 am" src="https://user-images.githubusercontent.com/3487289/46076638-fe1b3b80-c142-11e8-8da5-562a52ecd796.png">

Also matches cross over word elisions in the text search:

<img width="1182" alt="screen shot 2018-09-26 at 4 19 01 am" src="https://user-images.githubusercontent.com/3487289/46076745-4dfa0280-c143-11e8-9ed0-661ab04970da.png">

**Comment by emilianoricciardi** [comment]
Yes, I noticed this a few months ago. It certainly looks interesting...

**Comment by craigsapp** [comment]
The `@` signs are used to mark notes that will be highlighted.  Somehow the text is getting marked (and sometimes marked twice).  Once I track the cause of the problem it should be easy to fix.
